legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/h2Extension.pure

```diff
@@ -157,6 +157,14 @@ function <<access.private>> meta::relational::functions::sqlQueryToString::h2::c
       'cast( parseDateTime('+$params->at(0)+','+$dateFormat +') as date)';
 }
 
+function <<access.private>> meta::relational::functions::sqlQueryToString::h2::transformParseDateParams(params:String[*]):String[2]
```

Comment:
Think these tests should be organized according to new scheme by @vikask-gs.

@vikask-gs - Have we written down (in Legend), how new tests should be organized ?

Reply:
yes, communicating with Vaishnavi offline on this